contractcourt/breach_arbitrator.go

```diff
@@ -1636,12 +1639,28 @@ func taprootBriefcaseFromRetInfo(retInfo *retributionInfo) *taprootBriefcase {
 			//nolint:lll
 			tapCase.CtrlBlocks.Val.CommitSweepCtrlBlock = bo.signDesc.ControlBlock
 
+			bo.resolutionBlob.WhenSome(func(blob tlv.Blob) {
+				tapCase.SettledCommitBlob = tlv.SomeRecordT(
+					tlv.NewPrimitiveRecord[tlv.TlvType2](
+						blob,
+					),
+				)
+			})
+
 		// To spend the revoked output again, we'll store the same
 		// control block value as above, but in a different place.
 		case input.TaprootCommitmentRevoke:
 			//nolint:lll
 			tapCase.CtrlBlocks.Val.RevokeSweepCtrlBlock = bo.signDesc.ControlBlock
 
+			bo.resolutionBlob.WhenSome(func(blob tlv.Blob) {
+				tapCase.BreachedCommitBlob = tlv.SomeRecordT(
+					tlv.NewPrimitiveRecord[tlv.TlvType3](
+						blob,
+					),
+				)
+			})
+
 		// For spending the HTLC outputs, we'll store the first and
```

Comment:
I think we need to store the `resolutionBlob` for htlc's for the restart case? Also since I'm unaware of the custom channels commitment format, does the blob change if the htlc is brought to the second level?

Reply:
The blob doesn't change if the HTLC is brought to the second level. We can essentially pre-sign for it all. I have another PR that'll fill in some of the HTLC gaps here coming up.